src/main/java/pl/nogacz/snake/board/TwoPlayerBoard.java

```diff
@@ -0,0 +1,378 @@
+package pl.nogacz.snake.board;
```

Comment:
Burada her kod bloğu iki defa görüyoruz. Bence burada ek bir class yaratman gerekiyor. O classa bu kod bloklarından bir tane koyacaksın. Her bir player için bu classtan bir instance yaratacaksın. Daha sonra 


```
 if(!isEndGame )
            moveSnake();
        if(!isEndGame2)
            moveSnake2();
```

gibi bir kod yerine

```
if(!firstPlayer.isEndGame()) {
  firstPlayer.moveSnake();
}

if(!secondPlayer.isEndGame()) {
  secondPlayer.moveSnake();
}
```

gibi bir kodu kullanacaksin. Bu kodunu daha sadeleştirir ve tekrarı azaltır.

Reply:
Dediğiniz gibi yaptım hocam.  Birkaç tane kaldı ikili olanlar onlar da önceden yazılmış olan koda çok dokunmamak için öyle bıraktım.